qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/QuteSupportForTemplate.java

```diff
@@ -436,6 +437,7 @@ public ResolvedJavaTypeInfo getResolvedJavaType(QuteResolvedJavaTypeParams param
 		resolvedType.setMethods(methodsInfo);
 		resolvedType.setInvalidMethods(invalidMethods);
 		resolvedType.setExtendedTypes(extendedTypes);
+		resolvedType.setKind(JDTTypeUtils.getJavaTypeKind(type));

```

Comment:
Could you rename `setKind `method with `setJavaTypeKind `please. Do that for Qute JDT and Qute LS commons JavaTypeInfo class.